packages/headless/src/insight.index.ts

```diff
@@ -324,6 +325,15 @@ export type {
 } from './controllers/insight/generated-answer/headless-insight-interactive-citation';
 export {buildInteractiveCitation} from './controllers/insight/generated-answer/headless-insight-interactive-citation';
 export type {GeneratedAnswerStyle} from './features/generated-answer/generated-response-format';
+
+export type {
+  UserActionsState,
+  UserActionsProps,
+  UserActionsOptions,
+  UserActions,
+} from './controllers/insight/user-actions/headless-user-actions';
+export {buildUserActions} from './controllers/insight/user-actions/headless-user-actions';
+
```

Comment:
export type {
```suggestion
export type {
  UserActionsState,
  UserActionsProps,
  UserActionsOptions,
  UserActions,
  UserAction,
  UserSession,
} from './controllers/insight/user-actions/headless-user-actions';
export {buildUserActions} from './controllers/insight/user-actions/headless-user-actions';

```

We need `UserAction` and `UserSession` as well.